lib/ohai/plugins/darwin/passwd.rb

```diff
@@ -0,0 +1,9 @@
+Ohai.plugin(:Passwd) do
+  provides "loggedin_user"
+
+  collect_data(:darwin) do
+    unless loggedin_user
+      loggedin_user shell_out("stat -f \'\%Su\' /dev/console").stdout.chop
```

Comment:
We have a few questions here from our review:

1) Can more than 1 user be logged in at a time?
2) Can we expose log in type? What if the user is logged in remotely?